src/main/java/core/selenium/util/WebDriverMethod.java

```diff
@@ -116,6 +116,19 @@ public Boolean apply(final WebDriver driver) {
         });
     }
 
+    /**
+     * Gets the text in given web element.
+     * @param webDriver   is the explicitWait of Driver.
+     * @param locatorType is the type of the given locator.
+     * @param locator     is the locator of the web element.
+     * @return the text in web element.
+     */
+    public static String getTextInWebElement(final WebDriver webDriver, final String locatorType,
+                                             final String locator) {
+        fillMapAccordingLocator(locator);
+        return webDriver.findElement(byLocator.get(locatorType)).getAttribute("textContent");
```

Comment:
I think you have to use only the method getText() at the place of getAttribute("textContent").

Reply:
There are some times where the getText() method does not work as expected, that is why I used getAttribute() and it works consistently well.